boefjes/plugins/kat_censys/boefje.py

```diff
@@ -0,0 +1,22 @@
+from boefjes.plugins.models import Boefje, Normalizer
+
+Censys = Boefje(
+    id="censys",
+    name="Censys",
+    description=(
+        "NEEDS API KEY IN ENV - Use Censys to discover open ports, services, certificates"
+    ),
+    consumes={"IPAddressV4", "IPAddressV6"},
+    produces={"IPPort"},
```

Comment:
You might want to pass the `environment_keys` argument to specify the list of required environment keys for this Boefje, in this case `CENSYS_API_ID ` and `CENSYS_API_SECRET`.

For example:
```suggestion
    produces={"IPPort"},
    environment_keys=["CENSYS_API_ID", "CENSYS_API_SECRET"]
```